mobile/examples/java/hello_world/MainActivity.java

```diff
@@ -115,7 +119,8 @@ private void makeRequest() {
           String headerText = sb.toString();
 
           Log.d("MainActivity", message);
-          if (status == 301) {
+          if ((scheme == REQUEST_SCHEME_HTTP && status == 301) ||
+              (scheme == REQUEST_SCHEME_HTTPS && status == 200)) {
```

Comment:
What's the story with the different status codes per scheme here? Is this just how the server behaves? In any case, maybe we should add some comments? Or maybe just consider 3xx or 2xx a success irrespective of scheme?